Javascript/homework/responses/2.3-clock-design.md

```diff
@@ -4,5 +4,21 @@
 
 #### Task 3: Creating your clock
 
-Also define a clock class - this is where we will design a digital clock to print the time.
+After you're satisfied with the body of your page, define a block that styles elements in the `clock` class.
+
+![Screen Shot 2021-01-15 at 5.49.29 PM](/Users/danielleanzelone/Library/Application Support/typora-user-images/Screen Shot 2021-01-15 at 5.49.29 PM.png)
+
+First, set a width and a height for the box that's going to hold your time.  Add a background and a solid border using colors different from the screen's background so you can see what the box will look like.
+
+Once again, you'll want to set your `display` to `flex` and center your text using `align-items` and `justify-content` .  You need to do this again because these attributes in the body only centered the box on your page; repeating these lines in the clock section will center your text within the box.
```

Comment:
What is the display or flex? The question stands should we have students google this or explain every detail.